Sources/TootSDK/Models/Translation.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Translation.swift
+//
+//
+//  Created by Philip Chu on 1/30/24.
+//
+
+import Foundation
+
+public struct Translation: Codable {
+
+    /// HTML-encoded translated content of the status.
+    public var content: String
+    /// The translated spoiler warning of the status.
+    /// The Mastodon spec incorrectly lists this as spoiler_warning.
+    public var spoilerText: String
+    /// The translated poll options of the status.
+    /// The Mastodon spec incorrectly lists this as an array.
+    public var poll: Poll?
+    /// The translated media descriptions of the status
+    public var mediaAttachments: [MediaAttachment]
+    /// The language of the source text, as auto-detected by the machine translation provider.
+    /// ISO 639 language code
+    public var detectedSourceLanguage: String
+    /// The service that provided the machine translation.
+    public var provider: String
+    /// The target language
+    /// Not in the Mastodon spec, but in the Mastodon code
+    public var language: String
+
+    public struct MediaAttachment: Codable, Hashable, Identifiable, Sendable {
```

Comment:
I'd like the name here to better reflect that this is not an actual attachment but a translation for an existing media.

```suggestion
    public struct MediaAttachmentTranslation: Codable, Hashable, Identifiable, Sendable {
```